temporal/api/cloud/usage/v1/message.proto

```diff
@@ -19,6 +19,7 @@ message Summary {
     // Records grouped by namespace
     repeated RecordGroup record_groups = 3;
     // True if data for given time window is not fully available yet (e.g. delays)
+    // When true, records could be added in the future (until false) 
```

Comment:
"records could be added or updated for the given start_time and end_time" or something like that

Reply:
Updated: `When true, records for the given time range could still be added/updated in the future (until false)`